src/main/java/org/baeldung/Application.java

```diff
@@ -1,4 +1,4 @@
-package org.baeldung.spring;
```

Comment:
Sure, no worries, let's separate them and I'll have another look. 
Thanks for the cleanup work, 
Eugen.